cloudslang-cli/pom.xml

```diff
@@ -276,6 +278,15 @@
                                     <outputDirectory>${basedir}/target/cslang-cli/maven</outputDirectory>
                                     <destFileName>maven</destFileName>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.microfocus.itom.oo</groupId>
+                                    <artifactId>python</artifactId>
+                                    <version>${python}</version>
+                                    <classifier>win</classifier>
```

Comment:
shouldn't this be bin?

Reply:
removed the python from here, as we are documenting the steps to setup python.